test/metabase/api/user_test.clj

```diff
@@ -2,10 +2,21 @@
   "Tests for /api/user endpoints."
   (:require [clojure.test :refer :all]
             [metabase.http-client :as http]
-            [metabase.models.collection :as collection :refer [Collection]]
-            [metabase.models.permissions-group :as group :refer [PermissionsGroup]]
-            [metabase.models.permissions-group-membership :refer [PermissionsGroupMembership]]
-            [metabase.models.user :refer [User]]
+            [metabase.models
```

Comment:
is this change desired? at least the formatting. I've never seen this style before where each referred item gets its own line.

Reply:
Good catch. That's just what `cljr-refactor` did for me automatically (because the line would have been too long otherwise)... I'll go in and manually fix it

Reply:
fixed